ci/jenkins/test.sh

```diff
@@ -322,8 +322,12 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    govc snapshot.revert -vm ${WIN_NAME} win-initial
-    # If Windows VM fails to power on correctly in time, retry several times.
+    if [[ ${TESTCASE} =~ "containerd-ovs" ]]; then
```

Comment:
For jumper node, there is only one win-initial snapshot.

Reply:
For ovs containerization
In case of jumper node we are reverting to win-initial-snapshot and for other nodes , it is pristine-win-initial